src/test/scala/com/databricks/spark/avro/AvroSuite.scala

```diff
@@ -29,6 +29,15 @@ import org.scalatest.FunSuite
 /* Implicits */
 import TestSQLContext._
 
+private[avro] object DirectoryDeletion {
+  private[avro] def recursiveDelete(path: File): Unit = {
+    if (path.isDirectory) {
+      path.listFiles.foreach(recursiveDelete(_))
+    }
+    path.delete
```

Comment:
fyi: use of method invocation without parentheses is generally restricted to non-side-effect-y things, such as getters. Otherwise, it can hide method invocation versus value accessing. In general, when unsure, I would lean towards using parentheses.